scripts/build.js

```diff
@@ -28,6 +28,34 @@ function compareFiles(leftFile, rightFile) {
     return l.localeCompare(r, "en-US");
 }
 
+// Sort the examples based on the example-order defined in metadata.md. The missing example will be sort aphabetically at the end
+// Example:
+// metadata.md - activity-indicator
+// example-order: styling, basics
+function orderExamples (array, order, dirName) {
+    var customSortArray = [];
+    order.forEach(function(item){
+        var index = array.indexOf(dirName+"/"+item+"/article.md");
+        if(index >= 0){
+            customSortArray.push(array[index]);
+            array.splice(index, 1);
+        }
+    }); 
+    customSortArray.sort( function (first, second) {
```

Comment:
Is this sorting really necessary? The items in the `customSortArray` should already be pushed in the correct order.